frontend/src/views/PartnersPage/PartnersHome.vue

```diff
@@ -100,15 +100,15 @@
               for="select-sector"
               :class="{
                 'text-body-2': true,
-                'active-filter-label': filters.sectors.value && !!filters.sectors.value.length,
+                'active-filter-label': filters.sectorCategories.value && !!filters.sectorCategories.value.length,
               }"
             >
               Secteur d'activité
```

Comment:
I don't know -  do you think it would be better to change the label ? I'm wondering whether canteen managers would find it confusing to usually see one list of sectors, and here see something else. But then I don't know if 'Catégorie de secteur' is more clear. Thoughts?

Reply:
I think "secteur" is sufficiently vague for people to understand it's different depending on the context. "Catégorie de secteur" seems to me a bit more technical/internal to us